evalml/automl/automl_algorithm/default_algorithm.py

```diff
@@ -496,7 +500,11 @@ def _transform_parameters(self, pipeline, proposed_parameters):
                         )
             if name in self._pipeline_params:
                 for param_name, value in self._pipeline_params[name].items():
-                    component_parameters[param_name] = value
+                    if (
+                        param_name not in component_class.hyperparameter_ranges
```

Comment:
Make sure we set the pipeline_parameters as long as it doesn't appear in the hyperparameter ranges